### ISSUE TITLE
Missing images

### ISSUE THREAD

**Issue by honestleaf** [WFCD/warframe-items#78]
I just noticed that some images are missing. According to my finding, lots of the missing images are from `Resources.json` (espacailly those plain/vallis related resources).

**Comment by TobiTenno** [comment]
Please read the other issues before you make a new one. There's a already one about this #62

**Comment by honestleaf** [comment]
@TobiTenno Actually, `Resources.json` is not the only place where missing images occur. In `Misc.json` and `Glyphs.json`, there are some missing images too (ex. Trembera Essence and Saryn Prime Glyph), and that's why I open this issue.

If you have already awared of those problems, feel free to ignore this message.

**Comment by TobiTenno** [comment]
[Trembera](https://cdn.warframestat.us/img/relay-essence-earth.png) and [Saryn Prime Glyph](https://cdn.warframestat.us/img/saryn-prime-glyph%20-%20Lotus-Types-StoreItems-AvatarImages-ImageSarynPrimeBright.png) both show for me when I navigate to them. If you have other specifics, feel free to let us know, but these ones you've mentioned worked fine. I'm guessing you didn't url encode the latter of the two above?

**Comment by honestleaf** [comment]
Ah, yes, you are right on the Saryn Prime Glyph one, but I am still confused about the Trembera Essence. 

Here is what I got from `Glyphs.json`:
```
  {
    "uniqueName": "/Lotus/Types/Items/RelayRebuild/PhaseThreeEarthEssence",
    "name": "Trembera Essence",
    "description": "A highly purified mineral compound prized for its strength, resiliency, and resistance to heat. Widely used in the construction of Tenno relays.",
    "type": "Misc",
    "imageName": "trembera-essence.png",
    "category": "Misc",
    "tradable": false,
    "patchlogs": [
      {
        "name": "Mask of the Revenant: Hotfix 23.6.1",
        "date": "2018-08-30T23:04:15Z",
        "url": "https://forums.warframe.com/topic/1003947-mask-of-the-revenant-hotfix-2361/",
        "additions": "",
        "changes": "Added common resource beam FX to the Trembera Essence pickup.",
        "fixes": ""
      }
    ]
  }
```
As you can see the image name should be "trembera-essence.png". I'm wondering where did you get that "relay-essence-earth.png".

**Comment by TobiTenno** [comment]
Why is trembera essence even in Glyphs.json?

**Comment by TobiTenno** [comment]
It's in Misc.json, and yeah, it's matching up the wrong imageName, which means it's one of those that fell through the cracks with the last fix i did to try to cache image names.

**Comment by honestleaf** [comment]
It seems these items also have missing images:

Gear.json
* Force Specter
* Phase Specter
* Vapor Specter

Melee.json
* Tipedo
* Nikana
* Kronen

**Comment by Kaptard** [comment]
The items you listed have their images fixed in the [refactor/build](https://github.com/WFCD/warframe-items/tree/refactor/build) branch. Still need to verify that all others are there too, but it looks good from just skimming over it.

**Comment by Kaptard** [comment]
Merged and will be available on 1.294.0
